Taken from Oracle solaris-userland b83a793,
components/x11/xserver/xorg/patches/33-Xephyr-link.patch (xorg-server 21.1.24).

Solaris specific patch for clean build on sparc.

not suitable for upstream

--- a/hw/kdrive/src/kdrive.c    2019-02-13 07:20:23.524755470 -0800
+++ b/hw/kdrive/src/kdrive.c    2019-02-13 07:21:01.852569250 -0800
@@ -87,6 +87,12 @@
 const char *kdGlobalXkbVariant = NULL;
 const char *kdGlobalXkbOptions = NULL;

+#if ((defined(__sparc__) || defined(__sparc)) && defined (sun))
+int  num_total_disp_dev = 0;
+int  num_session_disp_dev = 0;
+char disp_dev_path[PATH_MAX];
+#endif
+
 void
 KdDisableScreen(ScreenPtr pScreen)
 {

